.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,33 @@
-## Testing
+## ℹ️ Description
 
 <!--
-What is needed to test the changes? e.g. specific cluster, service definitions
-How can one see the result of your work? e.g. configurations, URLs
+What is the change your PR does?
+Please add a short description here. (one sentence is enough)
 -->
 
-## Trade-offs
+- 🔖 JIRA: [DCOS-12345](https://jira.mesosphere.com/browse/DCOS-12345)
+  <!-- add things that reviewers need to know, e.g.
+- 🚫 This blocks #PRNumber
+- ⚠️ Component is not enabled yet (see testing)
```

Comment:
As you say, this should go under testing, shouldn't it?

Reply:
it might help reduce confusion stating it here - again: example and what is important in a particular PR decides the creator 🤷‍♂️

Reply:
Sorry, but that is not a good argument. I am sorry I did not take the time to write down in detail what I meant in this comment.

Reply:
maybe my wording also wasnt exact enough before, confusion might be the wrong word but it allows the PR creator to highlight certain _very important_ aspects. 

please, write it down.

Reply:
But what very important aspects are you referring to?

Reply:
you just highlighted one in #3087

Reply:
Okay, but this 

> ⚠️Please dont merge before Lily had a chance to see it 👍

is for me a really great example for a dependency. We can not merge it before lily took a look. Once she did that, we can add a checkmark or remove the line